config/secrets.yml

```diff
@@ -0,0 +1,18 @@
+default: &default
+    facebook_app_id: <%= ENV["FACEBOOK_APP_ID"] %>
+    facebook_app_secret: <%= ENV["FACEBOOK_APP_SECRET"] %>
```

Comment:
we don't need the facebook bits... in the next iteration we should only store the actual secrets, the rest can go as a standard config